components/src/dynamo/trtllm/publisher.py

```
@@ -954,26 +943,71 @@ def _handle_kv_event(self, event):
+                    normalized_event["block_hashes"], attention_dp_rank
+                )
+            return
+
+        self._publish_direct_kv_batch(attention_dp_rank, [normalized_event])
+
+    def _handle_kv_event_batch(self, events):
+        events_by_rank: dict[int, list[dict[str, Any]]] = {}
+        for event in events:
+            try:
+                normalized = self._normalize_kv_event(event)
+            except Exception as error:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)